MediaStream.js

```diff
@@ -81,6 +83,17 @@ export default class MediaStream extends EventTarget(MEDIA_STREAM_EVENTS) {
       } else {
           throw new TypeError(`invalid type: ${typeof arg}`);
       }
+
+      DeviceEventEmitter.addListener('mediaStreamTrackMuteChanged', ev => {
```

Comment:
mediaStreamTrackMuteChanged is also defined on the RTCPeerConnection.js. DeviceEventEmitter is also deprecated - events elsewhere use NativeEventEmitter (https://github.com/react-native-webrtc/react-native-webrtc/blob/master/EventEmitter.js -  https://github.com/react-native-webrtc/react-native-webrtc/blob/4c09877ac78e7f7081952d444c7ea8e05038cc45/RTCPeerConnection.js#L317